### DISCUSSION TITLE
Yürüyen merdiven sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#256] [Zeka Sorusu] [answered]
60 metre uzunluğundaki bir yürüyen merdiven, hareket etmeden duran bir kişiyi 60 saniyede yukarı çıkarmaktadır.
Yürüyen merdiven çalışmazken bu 60 metreyi 90 saniyede çıkıyorsunuz.
Yürüyen merdiven çalışırken, sizde yukarı doğru çıkarsanız kaç saniyede üst kata ulaşırsınız?

**Comment by adasue** [comment, marked as answer]
Eğer merdiven çalışmazken yukarı çıktığı mesafeyi saniyeye bölersek yani 60/90 yaptığımızda o kişinin saniyede 0,̅6 metre çıktığını buluyoruz. 60/60=1 ile de merdivenin kendisinin onu saniyede 1 metre yukarı çıkardığını buluyoruz. Eğer kişi merdiven hareket halindeyken yukarı doğru yürürse saniyede 1,̅6 metre çıkmış oluyor. 1,̅6'yı kesirli olarak gösterirsek de 15 bölü 9 oluyor. 60 metreyi 15 bölü 9'a bölersek de cevap **36** çıkıyor.
